workbenchfx-core/src/test/java/com/dlsc/workbenchfx/SingleModuleWorkbenchTest.java

```diff
@@ -0,0 +1,64 @@
+package com.dlsc.workbenchfx;
+
+import com.dlsc.workbenchfx.model.WorkbenchModule;
+import com.dlsc.workbenchfx.testing.MockPage;
+import com.dlsc.workbenchfx.testing.MockTab;
+import com.dlsc.workbenchfx.testing.MockTile;
+import javafx.collections.FXCollections;
+import javafx.scene.Node;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.stage.Stage;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockitoAnnotations;
+import org.testfx.api.FxRobot;
+import org.testfx.framework.junit5.ApplicationTest;
+import org.testfx.service.query.EmptyNodeQueryException;
+
+import static com.dlsc.workbenchfx.testing.MockFactory.createMockModule;
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Tests for {@link Workbench} in case only a single module is used
+ */
+public class SingleModuleWorkbenchTest extends ApplicationTest {
+    Workbench workbench;
+    private FxRobot robot;
+
+    @Override
+    public void start(Stage stage) {
+        MockitoAnnotations.initMocks(this);
+        robot = new FxRobot();
+
+        Node moduleNode = new Label("Module Content");
+        WorkbenchModule mockModule = createMockModule(
+                moduleNode, null, true, "Module 1", workbench,
+                FXCollections.observableArrayList(), FXCollections.observableArrayList()
+        );
+
+        workbench = Workbench.builder(
+                mockModule)
+                .tabFactory(MockTab::new)
+                .tileFactory(MockTile::new)
+                .pageFactory(MockPage::new)
+                .build();
+
+        Scene scene = new Scene(workbench, 100, 100);
+        stage.setScene(scene);
+        stage.show();
+    }
+
+    @Test
+    void moduleAutomaticallyOpened(){
+        assertEquals(workbench.getActiveModule(), workbench.getModules().get(0));
```

Comment:
Nit: doesn't really make a big difference, but you could also use `mockModule` here instead of `workbench.getModules().get(0)` if you declare `mockModule` like you do with `robot` currently.